.env.template

```diff
@@ -0,0 +1,6 @@
+# Flask server configuration.
```

Comment:
Minor, but now you have environment variables for your list ids it would be nice to include them in the `.env.template` file